### PR TITLE
sile 0.11.0

### PR DESCRIPTION
**Pull request by alerque**
SILE v0.11.0.

Also bumps two of the Lua dependencies.

Upstream release: https://github.com/sile-typesetter/sile/releases/tag/v0.11.0

### CODE REVIEW

**Review by SMillerDev — DISMISSED** [review]

**Comment by alerque** [comment]
Did something happen to the shell or version of autotools used by Homebrew? The `make` used in the last release had no problem with `.ONESHELL` and lines being continued in make targets with:

```
.ONESHELL:

target:
    echo "command one" ||
        echo "command two"
```

Now this time around it is throwing an unexpected end of file on the assumed continuation. Did an alternative POSIX shell get substituted in the last 6 months?

**Comment by SMillerDev** [comment]
Well, it's failing on ARM. Maybe zsh is the default there?

File: Formula/sile.rb [hunk 1]
```diff
@@ -25,6 +24,7 @@ class Sile < Formula
   depends_on "luarocks" => :build
   depends_on "pkg-config" => :build
   depends_on "fontconfig"
+  depends_on "git"
```
Comment (SMillerDev) [inline Formula/sile.rb:27]: Why does it depend on git now?

Comment (alerque) [reply Formula/sile.rb:27]: It actually has for a while, we just didn't document it or check at build time properly. SILE has an internal plugin system (package manager) that uses system calls to `git` to fetch plugins. You can use other functionality without it, but the package manager will die with an error.

This is mentioned in the release notes for v0.11.1.

Comment (SMillerDev) [reply Formula/sile.rb:27]: Maybe `uses_from_macos "git"` would be better then. Unless it needs a very specific version.

Comment (alerque) [reply Formula/sile.rb]: Sure, if that's a reliable way to have `git` in the path at run time that sounds fine. I just made the switch.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**